MSON Specification.md

```diff
@@ -439,12 +439,13 @@ A _Wildcard Type Name_ MAY only be used in a _[Type Specification][]_ for _[Memb
 #### 3.5.3 Type Attribute
 Defines extra attributes associated with the implementation of a type.
 
-- `required` - instance of this type is required
-- `optional` - instance of this type is optional (default)
-- `fixed`    - instance of this type structure and values are fixed
-- `nullable` - instance of this type *Value* MAY be unset (e.g. `null` or `nil`). `nullable` may only be used within properties of objects.
-- `sample`   - Alternate way to indicate a _[Value][]_ is a sample. See _[Sample][]_.
-- `default`  - Alternate way to indicate a _[Value][]_ is a default. See _[Default][]_.
+- `required`   - instance of this type is required
+- `optional`   - instance of this type is optional (default)
+- `fixed`      - instance of this type structure and values are fixed. This attribute is inheritable: all internal structures became fixed too.
+- `fixed-type` - instance of this type structure and values are fixed. This attribute works as fixed, but it is unheritable.
```

Comment:
I thought that `fixed` is a inheritable property, so if you set `fixed` attribute to some object or array all its values will be fixed to, so array will be completely static and if I use `fixed-type` - it makes static structure only on that level where `fixed-type` keyword was placed, so all internal structure will not be static.